src/utils/content.ts

```
@@ -0,0 +1,73 @@
+    return { contentType: 'document', mimeType };
+  }
+};
+
+export const isMediaMessage = (message: WAMessage): boolean => {
+  const mediaMessageTypes = [
+    'imageMessage',
+    'videoMessage',
+    'audioMessage',
+    'documentMessage',
+  ] as const;
+  const content = getContentType(message?.message!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)